Update to match L2

woodbe opened the pull request:
This is to match the L2 changes as well as fix found bugs in the naming of the attacks.

This adds:

- use of image and video as individual variants within the attacks 2 & 3
- removal of separate printer types, letting the lab determine whether they have a printer that meets the general requirements or just wants to use a service. Either is considered acceptable if the printer meets the listed parameters. Similarly the paper is either Matte or what the service provides, and so the types for paper supplies are also collapsed into a single item.
- addition of low and bright light on attack 4

For minimization, light conditions for attack 4 are only done using low and bright lights. Harsh shadows are, for the time being, left with L2.

woodbe committed:
Merge pull request #22 from biometricITC/Working

update from Greg

woodbe committed:
Proposed update to the Face toolbox

This is based on the planned changes to have the PAD-L2 face toolbox. This removed the full 3D while leaving the wearable mask (that is just taking a printed photo and wearing it like a mask)

This also tries to use the same structure from the fingerprint toolbox update in terms of naming/numbering.

woodbe committed:
Updates

Realized that I had moved sections that shouldn't have been moved, so this update puts them back.

woodbe committed:
Updates to incorporate changes from call

This update includes the following:

- adds a new environmental condition section to the inventory, defined following the description from the L2 update
- adds the new environmental conditions to the requirements in the attacks
- adjusts the overview to reference the new environment in the inventory (similar to referencing the tools)

woodbe committed:
Update L1-FACE_Toolbox_Inventory.adoc

Updated environment to include no harsh shadows

woodbe committed:
Merge pull request #23 from biometricITC/PAD-L1-Face

Proposed update to the Face toolbox

woodbe committed:
Update L1-FACE_Toolbox_Inventory.adoc

Corrected date for last update

woodbe committed:
Date fix on inventory file

Update L1-FACE_Toolbox_Inventory.adoc

woodbe committed:
Update README.adoc

Fixed broken links

woodbe committed:
updates

woodbe committed:
more updates to attacks

woodbe commented:
A question here as I'm considering how we have collapsed some of the testing by allowing for one type of printing. For the digital presentation, so we need all 4 combinations?

We have

1. phone capture X phone present
2. phone capture X big screen present
3. camera capture X phone present
4. camera capture X big screen present

I'm thinking that maybe only No.1 and No.4 are needed to be sufficient. Why would you take a high resolution camera and then present the image or video on a phone screen? And if you are taking the picture/video from the phone, wouldn't you be most likely to use that as the presentation method? I know you could get into taking it and then using a tablet to present, but I'm wondering if really testing the bottom and top are sufficient here.

This could be a bad trade, maybe artifacts from the phone going to the big screen could produce issues that let the image bypass. I don't know, but that seems to be the kind of thing that is more likely to be a L2 test where you are playing at the edges of the capabilities of the resolutions to incur issues that you aren't likely to be doing at L1 where you are probably trying for the best quality.

woodbe commented:
OK, I don't know what I was thinking about the changes to 2 & 3 for the attack types on Friday, but the next commit will revert those back since I literally just duplicated everything between those attacks across both.

woodbe committed:
fixed duplication

woodbe committed:
update test list

woodbe commented:
A question here. I have seen notes about testing for curved PAI being as simple as wrapping the photo around a cylinder that has roughly the diameter of a head so there is some level of 3D aspect to it. Nothing else, just that.

Do we want to add that to the FA1-04 attacks? That could actually be an alternative to wearing the mask, I'm OK either way, one or both, but I'm trying to figure out minimizing the number of tests. Wearing a photo has two potential benefits. One is say eye and lip wholes so you could have a real mouth and eyes showing through. The second is the possibility of the mask being better shaped to a real face, but that requires careful cutting and moving to get a shape created out of the paper. That could probably be level 1 because it's just time and effort with folding to get it right, but I don't know if we should use one or the other.

n-kai commented:
> A question here as I'm considering how we have collapsed some of the testing by allowing for one type of printing. For the digital presentation, so we need all 4 combinations?
> 
> We have
> 
> 1. phone capture X phone present
> 2. phone capture X big screen present
> 3. camera capture X phone present
> 4. camera capture X big screen present
> 

It seems that printed attack is more successful than digital attack (See *1) and we can remove some digital presentations. Mobile device may identify the non-matching background when a small screen is presented so only No.2 and No.4 may be needed to be sufficient.

*1) https://www.computerbild.de/artikel/cb-Tests-Handy-Gesichtserkennung-mit-Foto-geknackt-24910985.html

n-kai commented:
> A question here. I have seen notes about testing for curved PAI being as simple as wrapping the photo around a cylinder that has roughly the diameter of a head so there is some level of 3D aspect to it. Nothing else, just that.
> 
> Do we want to add that to the FA1-04 attacks? That could actually be an alternative to wearing the mask, I'm OK either way, one or both, but I'm trying to figure out minimizing the number of tests. Wearing a photo has two potential benefits. One is say eye and lip wholes so you could have a real mouth and eyes showing through. The second is the possibility of the mask being better shaped to a real face, but that requires careful cutting and moving to get a shape created out of the paper. That could probably be level 1 because it's just time and effort with folding to get it right, but I don't know if we should use one or the other.

I think that wearing a photo is enough for face L1 testing.

woodbe committed:
Update L1-FACE_Toolbox_Inventory.adoc

This commit is to close #27 for clarification.

woodbe commented:
Follow recommendation from https://github.com/biometricITC/Face-Toolbox/pull/29#issuecomment-3630932228 and then change the Inventory to use only a 4K or higher screen that is not a phone, and needs to be in the 20-30" diagonal range.

For simplicity, just stick with monitor, and not consider tablets (cost, and availability better).

woodbe commented:
Create a new attack specifically for cylinder wrapping since it is in current use.

woodbe committed:
update based on the last call

woodbe committed:
verification list update

woodbe commented:
I have added the cylinder to the inventory. I added it as variations on the printed flat photo attacks, since it was already mentioned as a possible variation (warping, which I left), as something to test with. The overall set is that this adds two more tests for the cylinder but removes the two for the phone screen presentation overall, so the result is a total of 12 (though it looks like we only had 8, so not sure why the original verification list didn't have some of the option). The end result is 2 curved prints and 2 masks with adjusted lighting conditions

woodbe committed:
standard artefact

woodbe commented:
I have added initial text for standard artefacts. We probably need to have something about expected cost, but I'm not sure. I know $2K was said a lot for decent ones and expensive ones are like $7K, but my understanding is that the more expensive ones are usually more like spoof masks for a real person as opposed to a generic face.

I have added this as a tool, not materials since it is something you would get once and keep since it isn't consumable.

I also added a little bit about this in the overview towards the end including the statement about not using standard artefact failures as the only measure to fail a component.

n-kai committed:
Enhance printing requirements for face images

Added requirements for maintaining aspect ratio and cropping the face image during printing.

n-kai reviewed (dismissed):
Thank you for your update. I have finshed my review and created the PR #30 for my proposed changes. I don't have any other comments.

woodbe committed:
Merge pull request #30 from biometricITC/n-kai-patch-1

Enhance printing requirements for face images
